lms/static/sass/views/_program-details.scss

```diff
@@ -301,12 +301,14 @@ $btn-color-primary: $primary-dark;
     display: flex;
     justify-content: flex-start;
     flex-direction: column;
+    margin-right: 48px;
```

Comment:
nit: we can replace `margin-right: 48px;` with bootstrap class => `mr-5`

Reply:
this class is already applied on the div so its better to add property in the class. no?

Reply:
Okay. You mean this element needs `48 + 48 = 96px` margin. So for better understanding and simplification, you can remove the bootstrap class and add the `margin-right` property with 96px.